sklearn/feature_extraction/text.py

```diff
@@ -979,6 +979,17 @@ class CountVectorizer(_VectorizerMixin, BaseEstimator):
     The ``stop_words_`` attribute can get large and increase the model size
     when pickling. This attribute is provided only for introspection and can
     be safely removed using delattr or set to None before pickling.
+    
+    ..versionchanged:: v0.22
+        Raise warning of parameter choide means the another parameter unused
```

Comment:
I am pretty sure we do not want to be that specific, though I don't know what would be a good rule of thumb to pick only the major changes.

Reply:
Ok I'm going to go through and remove all versionchanged tags, since we were only supposed to put in versionadded (according to the lead developer at our sprint). If we want to add in the versionchanged tags someone should put in a request for that with more details.

Reply:
Sounds good to me